### PR TITLE
Add PoolTogether funding model

### LINKED ISSUE #61: add a section on this token model
https://twitter.com/spencernoon/status/1188969044587831298

### PR DESCRIPTION
**Pull request by uivlis**
Fixes #61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -545,6 +546,29 @@ How can decentralized platforms up-end the revenue models for businesses that se
...
+    
+4. A second user believes very strongly that PoolTogether will be very successful, and capture a lot of Dai in its reserve. They come and purchase 1,000 PTG tokens for the cost of ~1,500 Dai (the bonding curve updates its price for total order size, as Uniswap does).
+    
+5. Because of this sale, the first user that purchase 1 PTG token at 1 Dai, can now come to the bonding curve, and submit their PTG token, and receive 1.50 Dai, for a .50 Dai profit.
+    
+6. But the first user also things that PoolTogether will be successful, so he doesn’t.
+    
+7. Hype generates about the token. Everyone wants some exposure to PoolTogether. The bonding curve takes in 500,000 Dai over the next few months, and issues 200,000 PTG tokens, now worth 2.5 Dai each.
+    
+8. The 500,000 Dai issued by the bonding curve goes to the Dai Reserve, is lent in compound, and receives ~675 Dai in interest rate every week (7% interest rate). This 675 Dai goes to the winning pool of the PoolTogether Lottery, which attracts many users and ticket sales, which adds funds to the Dai Reserve as well.
```
Comment (owocki) [inline README.md]: a visual might help people visualize how the systems grow together...

File: README.md [hunk 2]
```diff
@@ -545,6 +546,29 @@ How can decentralized platforms up-end the revenue models for businesses that se
...
+    
+3. Now, there is 1 Dai in the reserve. The new price for 1 PTG token is 1.01 Dai.
+    
+4. A second user believes very strongly that PoolTogether will be very successful, and capture a lot of Dai in its reserve. They come and purchase 1,000 PTG tokens for the cost of ~1,500 Dai (the bonding curve updates its price for total order size, as Uniswap does).
+    
+5. Because of this sale, the first user that purchase 1 PTG token at 1 Dai, can now come to the bonding curve, and submit their PTG token, and receive 1.50 Dai, for a .50 Dai profit.
+    
+6. But the first user also things that PoolTogether will be successful, so he doesn’t.
+    
+7. Hype generates about the token. Everyone wants some exposure to PoolTogether. The bonding curve takes in 500,000 Dai over the next few months, and issues 200,000 PTG tokens, now worth 2.5 Dai each.
```
Comment (owocki) [inline README.md]: > Hype 

this might be offputting to people

File: README.md [hunk 3]
```diff
@@ -545,6 +546,29 @@ How can decentralized platforms up-end the revenue models for businesses that se
...
+    
+8. The 500,000 Dai issued by the bonding curve goes to the Dai Reserve, is lent in compound, and receives ~675 Dai in interest rate every week (7% interest rate). This 675 Dai goes to the winning pool of the PoolTogether Lottery, which attracts many users and ticket sales, which adds funds to the Dai Reserve as well.
+
+For the complete description of the model visit https://medium.com/@TrustlessState/funding-model-for-pooltogether-and-other-money-lockup-apps-86b928900efa.
```
Comment (owocki) [inline README.md]: 9. the cycle repeats

**Comment by uivlis** [comment]
Hope it's good now.

**Pull request merged**